todoman/cli.py

```diff
@@ -148,6 +149,10 @@ def new(ctx, summary, list, todo_properties, interactive):
     if not todo.summary:
         raise click.UsageError('No SUMMARY specified')
 
+    default_due = int(ctx.obj['config']['main'].get('default_due', '24'))
```

Comment:
Yes, @untitaker is right. Ignore my comment, I posted it before I saw his response.

http://configobj.readthedocs.io/en/latest/validate.html#the-standard-functions